metadata-models/src/main/pegasus/com/linkedin/dataprocess/DataProcessInputOutput.pdl

```diff
@@ -0,0 +1,51 @@
+namespace com.linkedin.dataprocess
+
+import com.linkedin.common.DatasetUrn
+
+
+/**
+ * Information about the inputs and outputs of a Data process
+ */
+@Aspect = {
+  "name": "dataProcessInputOutput"
+}
+record DataProcessInputOutput {
```

Comment:
does it make sense to re-use fields across DataJobInputOutput and DataProcessInputOutput? 
If so, we can refactor out the common fields into a common record and then include it from both records.

Reply:
nice suggestion!
addressed in https://github.com/linkedin/datahub/pull/4114/commits/d30c11bbd227fc262f739831078b53578e4443e9